README.md

```diff
@@ -37,6 +37,50 @@ For an example of how to use this in an overall Modern Data Warehouse solution a
 
 For additional usage information, see [caching pipeline runs](#Caching-pipeline-runs).
 
+For testing storage event triggered pipelines, please use another fixture `adf_storage_event_triggered_pipeline_run`.
+
+```python
```

Comment:
Add bolded title called **"Storage Event: BlobCreated".**
Also, would be good to explicitly set the `blob_event_type="BlobCreated"` parameter even if it is optional just to make it clear. I think it is enough that you also mention this in the additional notes.

```python
this_run = adf_storage_event_triggered_pipeline_run(PIPELINE_NAME,
                                                        TRIGGER_NAME,
                                                        TRIGGER_FILE_PATH,
                                                        CONTAINER_NAME,
                                                        LOCAL_TRIGGER_FILE_PATH,
                                                        blob_event_type="BlobCreated") # Default value.
```